dashboard/app/helpers/application_helper.rb

```diff
@@ -69,6 +69,16 @@ def activity_css_class(user_level)
     end
   end
 
+  def page_complete_css_class(page_completed)
+    if page_completed == Game.PAGE_COMPLETED_PARTIAL
+      'attempted'
+    elsif page_completed == Game.PAGE_COMPLETED_PARTIAL
+      'perfect'
+    else
+      'not_tried'
+    end
+  end
```

Comment:
Same feedback as above: prefer to re-use existing [`ActivityConstants`](https://github.com/code-dot-org/code-dot-org/blob/6c595896a2ace7b0a6f84cc69264d7bb3d6b2b1c/lib/cdo/activity_constants.rb) / [`TestResults`](https://github.com/code-dot-org/code-dot-org/blob/8e49e8d56842a1fcaf891cd872c43d0343804f73/apps/src/constants.js#L24-L71) where possible.